.travis/script.sh

```diff
@@ -7,6 +7,7 @@ set -veuo pipefail
 
 # Lint code.
 flake8 --config flake8.cfg || exit 1
+black --check . || exit 1
```

Comment:
I suggest to add `--diff` here. This will show the corrections in the Travis log.